frontend/src/components/Contributions/ContributionForm.vue

```diff
@@ -104,11 +104,11 @@ export default {
   },
   methods: {
     submit() {
-      if (this.form.id) {
-        this.$emit('update-contribution', this.form)
-      } else {
-        this.$emit('set-contribution', this.form)
-      }
+      // not working for testing:
+      // this.$emit(this.form.id ? 'update-contribution' : 'set-contribution', this.form)
+      // works for testing:
+      // why ever, we have to make a spread '...', to evaluate the values it looks like: (I didn't find a solution in the test itmself)
+      this.$emit(this.form.id ? 'update-contribution' : 'set-contribution', { ...this.form })
```

Comment:
Now we have a solution to submit in tests and check for the emitted values. 🍀
YEAH !!!

Reply:
So what else can we do right or better?

Reply:
I did it now with @Mogge and @Elweyn

Reply:
Looks fine. Please remove the comments

Reply:
I reduced the comment to a minimum …